src/pages/index.js

```diff
@@ -5,29 +5,73 @@ import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupwithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards, config } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import { config } from "../utils/constants.js";
 
 const { formSelector } = config;
 
 //Button Elements
 
-const editProfileMOB = document.querySelector("#edit-profile-modal-open-button");
+const editProfileMOB = document.querySelector(
+  "#edit-profile-modal-open-button"
+);
 const addCardMOB = document.querySelector("#add-card-modal-open-button");
 const userNameInput = document.querySelector("#edit-profile-name-placeholder");
-const userJobInput = document.querySelector("#edit-profile-subtitle-placeholder");
+const userJobInput = document.querySelector(
+  "#edit-profile-subtitle-placeholder"
+);
+const profilePic = document.querySelector("#profile-photo");
+
+const editProfilePicModal = document.querySelector("#edit-pic-modal");
+const addCardModal = document.querySelector("#add-card-modal");
+const editProfileModal = document.querySelector("#edit-profile-modal");
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6730a6b1-094e-484e-b754-6ed1376cdeb2",
+    "Content-Type": "application/json",
+  },
+});
+
+const profileName = document.querySelector("#profile-name");
+const profileAbout = document.querySelector("#profile-subtitle");
 
+api.getUserInfo().then((userInfo) => {
+  profileName.textContent = userInfo.name;
+  profileAbout.textContent = userInfo.about;
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**